### ISSUE TITLE
Can't add a Pre-Request Script to Folders or Collections

### ISSUE THREAD

**Issue by solidcell** [a85/POSTMan-Chrome-Extension#12652]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

There is no "Edit" button as show in the docs here: https://learning.postman.com/docs/writing-scripts/pre-request-scripts/

### Steps To Reproduce

1. Tap the 3-dot button next to a folder or collection
2. There is no "Edit" option.

### Screenshots or Videos

https://www.loom.com/share/Postman-New-Folder-Loom-15-February-2024-bb702b5b618342049616f3cdb6c1dcb6

### Operating System

macOS

### Postman Version

10.23.0

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by solidcell** [comment]
I just created a brand new empty collection and folder and they both have "Edit". So it looks to only be a problem once I've added a single GraphQL request. Looks to be another issue that's GraphQL specific. Damn, a lot of Postman seems to take issue with GraphQL.

**Comment by ronork** [comment]
Hey @solidcell ! Thank you for reporting the issue.

Collections or folders with GraphQL request (and any other non-HTTP request) have certain limitations, this being one of them. We are tracking it internally and will share the updates when available.

**Comment by JohannesEmbrace** [comment]
@ronork @solidcell Yes, this is very annoying I'm just starting using GraphQL with Postman and would use the pre-request script to refresh my token before the request. For debug it's just calling an API endpoint so I didn't know how to use the standard Authorization options.

**Comment by Himanshu0781** [comment]
Hi everyone 👋

Thank you so much for taking the time to share your request. We really appreciate the thought and care you’ve put into describing the problem.

We’ve just rolled out a new feature. You can now add **collection and folder-level scripts** for collections that include GraphQL or gRPC requests! 🚀

Here’s a short video showing how it works: 

https://github.com/user-attachments/assets/e890e773-7595-400c-b444-8e5abfa743f6

We’d love to hear your feedback. Thanks for helping us make Postman better! 💪

Happy Debugging 💻 🐛
Himanshu Gupta
